fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -223,7 +221,7 @@
 
     // Enable use of legacy whole-system search parameters for the search request
     private final boolean legacyWholeSystemSearchParamsEnabled;
-    
+
```

Comment:
lots of whitespace changes on this file...to focus on actual changes you may want to hide the whitespace ones

Reply:
could end up with some merge issues with the payload offloading PR that's pending

Reply:
yeah, i need to go review that one.  if you want, we can hold on this one til thats merged...i don't mind handling conflicts